lib_lua/src/luaconf.h

```diff
@@ -515,7 +515,7 @@
 */
 #define LUAI_UACNUMBER	double
 
-#define LUA_NUMBER_PRECISION 3
+#define LUA_NUMBER_PRECISION 6
```

Comment:
> causing a usability issue with scripts.

I'm curious what usability issues you may be referring to.  Example?

> I think we can have the 6 precision if we modify the ftoa conversion to strip trailing zeros (and the decimal point if needed). That way, we can have the best of both worlds - full precision when we need it, and concise output when possible.

Not a bad idea.  Probably want a new method for it (like `ftoa_trim`) so that it doesn't break expectations for others using it.  Should be a pretty easy algorithm.

> Bonus: have lua use our common ftoa source

If you look at line 530 you can see that lua is already using our ftoa source code.